tests/provision/local/data/test.fmf

```diff
@@ -0,0 +1,2 @@
+test: ./test.sh
+framework: beakerlib
```

Comment:
```suggestion
/beakerlib:
    summary: Run helper scripts through beakerlib
    test: ./test.sh
    framework: beakerlib
/directly:
    summary: Run helper scripts directly
    test: ./directly.sh
```

I think @tcornell-bus has a good idea to exercise the helper scripts directly as well. Let's add one more test `directly.sh` which will call `tmt-report-result` directly, without `beakerlib`.